siliconcompiler/tools/surelog/surelog.py

```diff
@@ -57,6 +57,14 @@ def setup(chip):
     if no_write_cache == ['true']:
         options.append('-nowritecache')
 
+    no_info = chip.get('tool', tool, 'task', task, 'var', 'disable_info', step=step, index=index)
```

Comment:
Can you set them to false by default and add a help string so these options get documented

Reply:
If understand the other `var` options properly then it should already be off by default.
Added the docstring.